usmqe_tests/conftest.py

```diff
@@ -98,3 +100,20 @@ def valid_password(request):
     Return valid password string.
     """
     return request.param
+
+
+@pytest.fixture
+def os_info():
+    """
+    Return information from /etc/os-release file about current os distribution.
+    """
+    SSH = usmssh.get_ssh()
+    os_release = 'cat /etc/os-release'
+    f_content = SSH[
+        pytest.config.getini(
+            "usm_cluster_member")].run(
+                    os_release)[1].decode("utf-8").replace('"', '')
```

Comment:
It would be better to split this into few variables for better readability and maintenance.